ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContext.scala

```diff
@@ -70,8 +70,8 @@ private[kvutils] case class CommitContext(
     */
   def collectInputs[B, That](
       partialFunction: PartialFunction[(DamlStateKey, Option[DamlStateValue]), B]
-  )(implicit bf: BuildFrom[Map[DamlStateKey, Option[DamlStateValue]], B, That]): That = {
```

Comment:
There don’t bee to be `BuildFrom` instances for things like `BuildFrom[Map[DamlStateKey, Option[DamlStateValue]], Unit, Seq[Unit]]` on Scala 2.13. I don’t think we gain much from `BuildFrom` here so using `Factory` seemed easier than trying to have two versions one for the cases where `BuildFrom` works and  one where it doesn’t. It does make type inference slightly worse but it’s not too bad.

Reply:
I'm going to have to take your word (and the tests') that this is legit. Perhaps @miklos-da can weigh in?

I suggest you rename this variable from `bf` to `factory`.

Reply:
renamed in 0c1f641ea8935db50f0cf98660f41ad33bd2bd85. I’m happy to go for something else if you have a suggestion.

Reply:
I don't have a better suggestion. `Factory` should be preferred in Scala 2.13 if you don't want to rebuild with the best matching type for an existing source collection (but would like to leave that open to the consumer). It makes sense in this instance as the user knows e.g. what `partialFunction` will be producing and how selective it is.